tests/CppUTestExt/IEEE754PluginTest_c.c

```diff
@@ -25,6 +25,10 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <CppUTest/CppUTestConfig.h>
+
+#ifdef CPPUTEST_HAVE_FENV
```

Comment:
It's because of the INFINITE define later on.  For at least some versions of Visual C++, INIFINITE isn't defined and so I was getting compiler errors.  This was the quickest way I thought of to avoid that issue (the file doesn't get used unless FENV is active), but the correct solution probably would have been to just block off the one function if INFINITE is missing.